kombu/transport/SQS.py

```diff
@@ -490,7 +491,11 @@ def _optional_b64_decode(byte_string):
 
     def _message_to_python(self, message, queue_name, q_url):
         body = self._optional_b64_decode(message['Body'].encode())
-        payload = loads(bytes_to_str(body))
+        try:
```

Comment:
i like the try... except block, but if the client is not node, will this be the same?

Reply:
I added some more tests for this and refactored the method. The test cases have the following:

```    
@pytest.mark.parametrize('body', [
        '',
        'True',
        'False',
        '[]',
        '{}',
        '1',
        '1.0',
        'This is the body as a string'
    ])
    def test_messages_to_python_body_as_a_string(self, body):
```

I *think* we are safe, as the message would be a Kombu Message, and that defines the body as a string.
```
class Message:
    """Base class for received messages.

    Keyword Arguments:
    -----------------
        channel (ChannelT): If message was received, this should be the
            channel that the message was received on.

        body (str): Message body.
...
```